### ISSUE TITLE
How up to date is this library?

### ISSUE THREAD

**Issue by ethindp** [etcimon/botan#70]
How up to date does this library go in terms of upstream Botan version? Does it support Botan 3 (latest version is 3.1.1) or is it still on Botan 2?

**Comment by ethindp** [comment]
@etcimon I know, Botan is definitely a good library and I imagine that D could only add to it. Are there any plans to synchronize with Botan 3?

**Comment by ethindp** [comment]
I ask particularly since Botan 3 has TLS 1.3 support, among [a variety of other changes](https://botan.randombit.net/news.html).

**Comment by etcimon** [comment]
Yes, TLS 1.3 is the more interesting change. I've looked into it and it would require creating an interface out of the current objects and 2 implementations, for 1.2 and 1.3
